tests/src/CLRTest.Execute.targets

```diff
@@ -96,8 +97,8 @@ This file contains the logic for providing Execution Script generation.
   -->
   
   <ItemGroup>
-    <ExecutionScriptKind Include="Batch" />
-    <ExecutionScriptKind Include="Bash" />
+    <ExecutionScriptKind Include="Batch" Condition="'$(BuildOS)' == 'Windows_NT'"/>
+    <ExecutionScriptKind Include="Bash" Condition="'$(BuildOS)' != 'Windows_NT'"/>
```

Comment:
Does this mean the Bash scripts won't be created in Windows builds anymore?  If so, that seems bad until all of the existing workflows that rely on copying tests from Windows builds to Unix are updated.  Is this affected by the change of BuildOS default to AnyOS?  If so, that also seems like this may not be doing what we want.  Not sure.